LondonBritishLibrary/orient/BLorient801.xml

```diff
@@ -30,22 +31,76 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 367</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
-                        
-                        
-                        
-                    </msContents>
-                    
-                    
-                    
-                    
-                    
-                    
+
+<msItem xml:id="ms_i1">
+<locus from="2r"/>
+<title type="complete" ref="LIT1395Fethan"/>
+</msItem>
```

Comment:
This msItem should not be closed here, because the following msItems contain its parts, so it should be closed only at the end of the entire work.